tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -264,6 +271,45 @@ RHODS Dahsboard Pod Should Contain OauthProxy Container
         List Should Contain Value    ${container_name}    oauth-proxy
     END
 
+Is Environment Staging
+    [Documentation]    Return true if environment is staging otherwise fails
+    ${list_of_links} =    Get Links From Switcher
+    Should Contain    ${list_of_links}[0]   devshift.org
+
+Check Application Switcher Links To Openshift Cluster Manager
+    ${ocm_staging_link} =    Set Variable    https://qaprodauth.cloud.redhat.com/openshift/details/
+    ${cluster_id} =    Get Cluster ID
+    ${cluster_id} =    Remove String    ${cluster_id}    "
+    ${list_of_links} =    Get Links From Switcher
+    ${status}    Run Keyword And Return Status    Is Environment Staging
+    IF    "${status}" == "True"
+        Check HTTP Status Code    ${ocm_staging_link}${cluster_id}
+        Should Be Equal    ${list_of_links}[1]    ${ocm_staging_link}${cluster_id}
```

Comment:
I think this is gonna fail. If it's stage env, the link to OCM from the Switcher will not contain qaprodauth

Reply:
so, replace it with a check which fits the case. The link in the switcher will be equal to the production link